### ISSUE TITLE
Numpy 1.12.0 errors

### ISSUE THREAD

**Issue by CEKrause** [GeoscienceAustralia/tcrm#32]
Errors only appeared with numpy-1.12.0. 
When running stochastic simulation:
```
TrackGenerator.run(configFile, status)
File "/short/w85/cek156/tcrm/TrackGenerator/TrackGenerator.py", line 1871, in run
tracks = tg.generateTracks(sim.ntracks, sim.index)
File "/short/w85/cek156/tcrm/TrackGenerator/TrackGenerator.py", line 661, in generateTracks
data = np.core.records.fromarrays(data, dtype=track_dtype)
File "/home/156/cek156/.local/lib/python2.7/site-packages/numpy/core/records.py", line 605, in fromarrays
raise ValueError("mismatch between the number of fields "
ValueError: mismatch between the number of fields and the number of arrays
```
I thought this was an error in my personal environment (and still possibly is), but it seems to be related to the newer version of numpy (was previously using 1.8.0).

**Comment by wcarthur** [comment]
If you can create a virtual environment in which you install different numpy versions, you should be able to test if this is related to changes in that module.

**Comment by CEKrause** [comment]
Code error in TrackGenerator.py, not a numpy error.

Issue was in the way beta was implemented in the v3.0_beta branch.
